experimental/traffic-portal/src/app/core/statuses/status-details/status-details.module.ts

```diff
@@ -0,0 +1,36 @@
+import { NgModule } from '@angular/core';
```

Comment:
This file is missing a license header. See other typescript files for a comment block at the top that you can copy.